man/systemd.exec.xml

```diff
@@ -930,6 +930,37 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         <varname>SystemCallFilter=~@mount</varname>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TemporaryFileSystem=</varname></term>
+
+        <listitem><para>Takes a space-separated list of mount points for temporary file system (tmpfs). If sets, new
+        file system namespace for executed processes is set up, and temporary file system is mounted on each mount poiunt.
+        This option may be specified more than once, in which case temporary file system is mounted on all listed mount points.
+        If the empty string is assigned to this option, the list is reset, and all prior assignments have no effect.
+        Each mount point may be prefixed with <literal>-</literal>, in which case it will be ignored when it does not exist.
+        Each mount point may optionally be suffixed with a colon (<literal>:</literal>) and mount option such as
+        <literal>size=10%</literal> or <literal>ro</literal>. If no mount option is specified, then the tmpfs is mounted
+        with <literal>mode=0755</literal>. Note that each tmpfs enables <literal>nodev</literal> and <literal>strictatime</literal>
+        options, that is, each tmpfs is mounted with <literal>MS_NODEV|MS_STRICTATIME</literal> mount flags.
+        This can be disabled by explicitly specifying the mount option <literal>dev,nostrictatime</literal>.</para>
+
+        <para>This is useful to hide files or directories not relevant to the processes invoked by the unit, while necessary
+        files or directories can be still accessed by combining with <varname>BindPaths=</varname> or
+        <varname>BindReadOnlyPaths=</varname>. See the example below.</para>
+
+        <para>Example: if a unit has the following,
+        <programlisting>TemporaryFileSystem=/var
```

Comment:
I was trying to say that we should have an example which shows a tmpfs with `:ro`, and that it still works to have BindReadOnlyPaths= or whatever on paths inside it.  `:ro` is the more general configuration, for this use case, because it will work the same for `root`: it will stop you filling up RAM/swap space with files when you didn't realize the app was still configured to write to one of the tmpfs directories.